### PR TITLE
Update deploy-manage-removable-storage-intune.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation

fixes #https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10321

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/deploy-manage-removable-storage-intune.md [hunk 1]
```diff
@@ -45,11 +45,12 @@ For policy deployment in Intune, the account must have permissions to create, ed
 
 ## Deploy Removable Storage Access Control by using Intune OMA-URI
 
-Go to Microsoft Endpoint Manager admin center (<https://endpoint.microsoft.com/>) > **Devices** > **Create profile** > **Platform: Windows 10 and later, Profile type: Templates** > Custom**.
+Go to Microsoft Endpoint Manager admin center (<https://endpoint.microsoft.com/>) > **Devices** > **Configuration profiles** > **Create profile** > **Platform: Windows 10 and later, Profile type: Templates** > **Custom** > **Create**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/deploy-manage-removable-storage-intune.md]:
```suggestion
Go to Microsoft Endpoint Manager admin center (<https://endpoint.microsoft.com/>) > **Devices** > **Configuration profiles** > **Create profile** > **Platform: Windows 10 and later, Profile type: Templates** > **Custom** > **Create**.
```

File: microsoft-365/security/defender-endpoint/deploy-manage-removable-storage-intune.md [hunk 2]
```diff
@@ -45,11 +45,12 @@ For policy deployment in Intune, the account must have permissions to create, ed
...
 
 1. Enable or Disable Device control as follows:
 
-   - Under **Custom** > **Configuration settings**, select **Add**.
+   - Under **Custom** > Enter the **Name**, **Description** and click on **Next**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/deploy-manage-removable-storage-intune.md]:
```suggestion
   - Under **Custom**, enter the **Name** and **Description** and select **Next**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Pleas merge this PR. Thanks

**Comment by yogkumgit** [comment]
@beccarobins Please merge this PR or review and Sign off. Thanks

**Pull request merged**